helm/cfgov/values.yaml

```
@@ -179,6 +179,14 @@ cronJobs:
       - "clearsessions"
     restartPolicy: OnFailure
 
+  - name: "publishPages"
+    schedule: "* * * * *"
+    command:
+      - "publish_scheduled_pages"
+    args:
+      - "publishPages"
```

The management command here is [`publish_scheduled_pages`](https://docs.wagtail.org/en/stable/reference/management_commands.html#publish-scheduled-pages).

```suggestion
      - "publish_scheduled_pages"
```